src/main/java/com/emt/entity/Match.java

```diff
@@ -31,4 +32,6 @@ public class Match {
   private Player loser;
 
   @NotNull private Instant createdAt;
+
+  @Embedded private EloRatingChange ratingChange;
```

Comment:
Avoid using an embedded entity here; you can keep the separation in the response model only